partition/roles/sonic/templates/metal.yaml.j2

```diff
@@ -107,14 +107,16 @@ PORT:
     {% if 'parent_port' in running_cfg %}
     parent_port: {{ running_cfg.parent_port }}
     {% endif %}
-    {% if sonic_ports_dict[name] is defined %}
-    {% set port = sonic_ports_dict[name] %}
     admin_status: up
```

Comment:
Why always setting admin_status to up? Why not similar to fec, mtu, and speed?

Reply:
We could introduce another variable for the admin_status, but that would possibly involve a lot of changes to the existing deployments, right?